peer/brontide.go

```diff
@@ -3158,9 +3242,13 @@ func (p *Brontide) handleLocalCloseReq(req *htlcswitch.ChanClose) {
 				return
 			}
 		}
-
+		addr, err := p.addrWithInternalKey(deliveryScript).Unpack()
+		if err != nil {
+			req.Err <- fmt.Errorf("unable to parse addr: %w", err)
+			return
+		}
```

Comment:
In every other error handling statement we log the error. We might want to do that here also.